hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -983,10 +974,6 @@ public void loadAll(Set<K> keys, boolean replaceExistingValues) {
 
         Collection<Data> dataKeys = CollectionUtil.objectToDataCollection(keys, getSerializationService());
         loadAllInternal(replaceExistingValues, dataKeys);
-
-        if (replaceExistingValues) {
```

Comment:
Could not find a test for this, so I wrote one: https://github.com/hazelcast/hazelcast/pull/8869/files#diff-ec815c78e843f777f47642c555a0d9b2R147

:+1: